homeassistant/components/tuya/number.py

```diff
@@ -69,12 +69,16 @@
     "cwwsq": (
         NumberEntityDescription(
             key=DPCode.MANUAL_FEED,
-            translation_key="feed",
+            translation_key="manual_feed",
```

Comment:
I don't think we have this translation?

Reply:
Where do I add translation? There also exist quick feed so it should not be called feed i think.